Gemfile.lock

```diff
@@ -653,7 +653,7 @@ GEM
     recaptcha (5.17.0)
     redcarpet (3.6.0)
     redis (4.8.1)
-    redis-actionpack (5.3.0)
+    redis-actionpack (5.4.0)
```

Comment:
Updated gem version since it is failing the build while running `bundle install`